source/reference/minio-mc/mc-support-callhome.rst

```diff
@@ -0,0 +1,138 @@
+=======================
+``mc support callhome``
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs disable
+.. mc:: mc support logs enable
+.. mc:: mc support logs status
+   
+.. mc:: mc support callhome
+
+Description
+-----------
+
+The :mc-cmd:`mc support callhome` command allows the enabling or disabling of reports from a deployment to |SUBNET|.
+As with other ``mc support`` commands, sending of reports requires an active SUBNET subscription.
+
+When enabled, MinIO can send logs to SUBNET in real time, diagnostics every 24 hours, or both.
+
+This functionality is disabled by default, even for clusters registered with SUBNET.
```

Comment:
Could simplify as just "MinIO disables this functionality by default, regardless of registration status. You must explicitly enable it to begin real time log upload". Also removes the passive.

Reply:
👍